raster3d/r3.info/main.c

```diff
@@ -48,7 +48,7 @@ int main(int argc, char *argv[])
 {
     const char *mapset;
     char *line = NULL;
-    char tmp1[TMP_LENGTH], tmp2[TMP_LENGTH], tmp3[TMP_LENGTH];
+    char tmp1[TMP_LENGTH] = "", tmp2[TMP_LENGTH] = "", tmp3[TMP_LENGTH] = "";
```

Comment:
I don't remember enough (without reading any references) the exact syntax and pitfalls to avoid when initializing C char arrays. I'm leaving the review to someone else.

Reply:
This approach would work just fine.